.nais/kafka/friskmelding-til-arbeidsformidling-vedtak-fattet.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  annotations:
+    dcat.data.nav.no/title: "Vedtak relatert til friskmelding til arbeidsformidling for sykmeldte personer"
+    dcat.data.nav.no/description: >-
+      Topic inneholder informasjon om vedtak relatert til friskmelding til arbeidsformidling for sykmeldte personer.
+  name: friskmelding-til-arbeidsformidling-vedtak-fattet
```

Comment:
Kunne man kanskje bare brukt `isfrisktilarbeid` i stedet for `friskmelding-til-arbeidsformidling`, så blir den ikke så lang? 🤔

Reply:
Litt enig med John-Martin 😉